e2e/run/run.go

```diff
@@ -206,5 +306,8 @@ func E2ETests(env e2e.TestEnv) testhelper.Tests {
 		"inaccessible home":    c.issue409,
 		"passphrase encrypted": c.testRunPassphraseEncrypted,
 		"PEM encrypted":        c.testRunPEMEncrypted,
+		"user overlayfs":       c.testUserOverlayfs,
+		"squashfuse mount":     c.testSquashfuseMount,
+		"ext3 mount":           c.testExt3Mount,
```

Comment:
I think the tests are good, but let's be more consistent in the naming.  I suggest
```
"fuse overlayfs":         c.testFuseOverlayfs,
"fuse squash mount": c.testFuseSquashMount,
"fuse ext3 mount":     c.testFuseExt3Mount,
```

Reply:
resolved and squashed all commits into one
https://github.com/apptainer/apptainer/pull/615/commits/017cb82de2a3167b83e98eeb19e156cc52939e7c